README.md

```diff
@@ -37,6 +37,8 @@ and then use them in your components. Examples for each hook and utility can be
 
 `useNetworkStatus` React hook for getting network status (effective connection type)
 
+This hook accepts an optional `initialEffectiveConnectionType` string argument, which can be used to provide a `effectiveConnectionType` state value when the user's browser does not support the relevant [NetworkInformation API](https://wicg.github.io/netinfo/). Passing an initial value can also prove useful for server-side rendering, where the developer can pass an [ECT Client Hint](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/client-hints#ect) to detect the effective network connection type.
+
```

Comment:
These `initial<Values>` docs are really great. Thanks for also linking up to the corresponding Client Hints reference. One suggestion for each of Network/SaveData/Memory: perhaps we can move the initial values paragraph to after the primary code-snippet and include a short 1-2 line example of the initial value being set?

Reply:
No problem, will do 👍